### ISSUE TITLE
why there is no activation function in outconv ?

### ISSUE THREAD

**Issue by kobe233333** [milesial/Pytorch-UNet#181]
class OutConv(nn.Module):
    def __init__(self, in_channels, out_channels):
        super(OutConv, self).__init__()
        self.conv = nn.Conv2d(in_channels, out_channels, kernel_size=1)

    def forward(self, x):
        return self.conv(x)

**Comment by milesial** [comment]
For the last layer we need raw logits so we can later apply sigmoid or softmax (depending on the number of classes) on them.

**Comment by kobe233333** [comment]
> For the last layer we need raw logits so we can later apply sigmoid or softmax (depending on the number of classes) on them.

thanks,i got it!
